scene/resources/mesh.cpp

```diff
@@ -224,8 +224,12 @@ Ref<TriangleMesh> Mesh::generate_triangle_mesh() const {
 
 			if (primitive == PRIMITIVE_TRIANGLES) {
 				for (int j = 0; j < ic; j++) {
+					ERR_FAIL_COND_V_MSG(indices.size() > j,Ref<TriangleMesh>(),"invalid index");
```

Comment:
Shouldn't this be the other way around? Preemptively indices.size() should always be larger than j when j is 0

Reply:
yep I revised this in my own internal version, you are correct